src/generic_core/remote-instance.ts

```diff
@@ -229,9 +232,12 @@ module Core {
               this.rtcToNet_ = null;
             });
           }
-          // TODO: signalFromRemote needs to get converted into a
-          // WebRtc.SignallingMessage. This probably doesn't actually work right
-          // now.
+          if (!this.rtcToNet_) {
```

Comment:
If the peer sends messages out of order (e.g. sends a CANDIDATE before the initial OFFER).  We saw this happen in https://github.com/uProxy/uproxy/issues/784